src/com/hrabrov/introduction_to_java/algorithmization/one_dimensional_array/OneDimensionalArray.java

```diff
@@ -0,0 +1,45 @@
+package com.hrabrov.introduction_to_java.algorithmization.one_dimensional_array;
+
+import java.util.Arrays;
+
+public class OneDimensionalArray {
+    public static void run() {
+        int resultOfTask01 = Task01.sumMultipleNumber(new int[]{3, 4, 5, 6, 8}, 3);
```

Comment:
Create an array first, then pass it as param. Your way is unreadable.
Nitpick: you need to add a space between `[]` and `{...}`